front/src/components/Table/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+} from "@material-ui/core";
+
+import { selectBooksBasket } from "features/books/booksSlice";
+
+import { useStyles, InfoElement } from "./styles";
+
+function createData(
+  name: string,
+  calories: number,
+  fat: number,
+  carbs: number,
+  protein: number
+) {
+  return { name, calories, fat, carbs, protein };
+}
+
+interface Book {
+  author: string;
+  cover_url: string;
+  currency: string;
+  id: number;
+  pages: number;
+  price: number;
+  title: string;
+  value: number;
+}
+
+export default function BasicTable() {
+  const basket = useSelector(selectBooksBasket);
+  console.log(basket);
+
+  const classes = useStyles();
+
+  const newRows = basket.map((book: Book) => {
```

Comment:
getNewRows - to nie zmienna, a metoda. Imho lepiej byłoby zrobić 

basket.map((book: Book) => ({
author: book.author,
      title: book.title,
      pages: book.pages,
      price: book.price,
      currency: book.currency,
      cover_url: book.cover_url,
      id: book.id,
})

Troche bardziej przejrzyscie i funkcyjnie

Reply:
A najlepiej zrobić selektor, np. selectNormalizedBooks, który to zwróci. Bo rozumiem, że to jest dlatego, że book zwracany z backendu ma jeszcze więcej pól, których tu nie potrzebujesz

Reply:
mogłem od razu mapować basket nie wiem czemu tworzyłem zmienną aby zwrócić z niej to samo co było zwracane z basket... Usunąłem,
Done.